libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.service.ts

```diff
@@ -362,7 +383,23 @@ export class ParentalLeaveService extends BaseTemplateApiService {
     const { applicationFundId } = getApplicationExternalData(
       application.externalData,
     )
-
+    const attachmentsFiles = (application.answers.fileUpload as unknown) as {
```

Comment:
Can you add this to getApplicationAnswers in parentalLeaveUtils?

Reply:
Yes def.